docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,15 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms`>> and model 
+plot is not enabled for the {anomaly-job}, then neither the **Single Metric 
+Viewer** nor the **Anomaly Explorer** in {kib} can plot and display an anomaly 
+chart for the job. Single Metric Viewer doesn't support configurations where 
```

Comment:
Sorry, this needs more editing as this issue affects the charts in both the Anomaly Explorer and Single Metric Viewer. I'm not sure how best to phrase this - charts are displayed, but the values plotted in the charts don't match up unless the job bucket span is the same as the interval used for the date_histogram aggregation. For example, here the bucket_span is `30m` and the date_histogram aggregation interval is `15m`. The result is the data in the chart is twice the actual value:

![image](https://user-images.githubusercontent.com/7405507/99655168-44cefc80-2a53-11eb-96ef-89d23f2ed989.png)